packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1459,20 +1459,29 @@
 	"contextual.credentials.sharing.unavailable.description": "You can share credentials with others when you upgrade your plan.",
-	"contextual.credentials.sharing.unavailable.description.cloud.upgrade": "You can share credentials with others when you upgrade your plan.",
+	"contextual.credentials.sharing.unavailable.description.cloud": "You can share credentials with others when you upgrade your plan.",
+	"contextual.credentials.sharing.unavailable.description.desktop": "Sharing features are available on selected Cloud plans",
 	"contextual.credentials.sharing.unavailable.button": "View plans",
-	"contextual.credentials.sharing.unavailable.button.cloud.upgrade": "Upgrade now",
+	"contextual.credentials.sharing.unavailable.button.cloud": "Upgrade now",
+	"contextual.credentials.sharing.unavailable.button.desktop": "View plans",
 
 	"contextual.workflows.sharing.title": "Sharing",
 	"contextual.workflows.sharing.unavailable.title": "Sharing",
-	"contextual.workflows.sharing.unavailable.title.cloud.upgrade": "Upgrade to add users",
+	"contextual.workflows.sharing.unavailable.title.cloud": "Upgrade to collaborate",
+	"contextual.workflows.sharing.unavailable.title.desktop": "Upgrade to n8n Cloud to collaborate",
 	"contextual.workflows.sharing.unavailable.description.modal": "You can collaborate with others on workflows when you upgrade your plan.",
-	"contextual.workflows.sharing.unavailable.description.modal.cloud.upgrade": "You can collaborate with others on workflows when you upgrade your plan.",
+	"contextual.workflows.sharing.unavailable.description.modal.cloud": "You can collaborate with others on workflows when you upgrade your plan.",
+	"contextual.workflows.sharing.unavailable.description.modal.desktop": "Upgrade to n8n Cloud to collaborate on workflows: sharing features are available on selected plans.",
 	"contextual.workflows.sharing.unavailable.description.tooltip": "You can collaborate with others on workflows when you upgrade your plan. {action}",
-	"contextual.workflows.sharing.unavailable.description.tooltip.cloud.upgrade": "You can collaborate with others on workflows when you upgrade your plan. {action}",
+	"contextual.workflows.sharing.unavailable.description.tooltip.cloud": "You can collaborate with others on workflows when you upgrade your plan. {action}",
+	"contextual.workflows.sharing.unavailable.description.tooltip.desktop": "Upgrade to n8n Cloud to collaborate on workflows: sharing features are available on selected plans. {action}",
 	"contextual.workflows.sharing.unavailable.button": "View plans",
-	"contextual.workflows.sharing.unavailable.button.cloud.upgrade": "Upgrade now",
-	"contextual.workflows.sharing.unavailable.linkUrl": "https://subscription.n8n.io/"
+	"contextual.workflows.sharing.unavailable.button.cloud": "Upgrade now",
+	"contextual.workflows.sharing.unavailable.button.desktop": "View plans",
+	"contextual.upgradeLinkUrl": "https://subscription.n8n.io/",
```

Comment:
I'm not sure if this matters here but on the Usage page the subscription app is environment specific.
See [here](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/stores/usage.ts#L53)

Reply:
The subscription link gets replaced with the correct one (including the instance id) at runtime, this is just a placeholder that we check against.